### DISCUSSION TITLE
Init state warning:  Some mods and modpacks may require Java 8

### DISCUSSION THREAD

**Discussion by Navio1430** [itzg/docker-minecraft-server#2708] [Q&A] [answered]
Hello,

I've got a problem where the warning message is showing on each server startup. I'm using a custom jar which is one of the paper's forks.  The warning is disappearing after couple of seconds. But definitely those seconds make a difference for me when it comes to testing. 

Also where can I disable the [mc-image-helper]?

regards

![image](https://github.com/itzg/docker-minecraft-server/assets/73886336/bdf624d4-8cb3-4632-9a50-69c0ccbe4f7c)

**Comment by itzg** [comment, marked as answer]
I'll create an issue to remove that warning and definitely the 5 second pause that it does. That check used to be a bit more sensible when the server choices were smaller and modpack with their selected loaders weren't involved.

**Reply by itzg** [reply]
https://github.com/itzg/docker-minecraft-server/issues/2709

**Reply by itzg** [reply]
If you re-pull the image the warning and delay should now be gone.

**Reply by Navio1430** [reply]
Yup, everything works fine now. Thanks.
